pipeline/frontend/yaml/compiler/convert.go

```diff
@@ -184,3 +184,7 @@ func (c *Compiler) createProcess(name string, container *yaml.Container, section
 		IpcMode:      ipcMode,
 	}
 }
+
+func (c *Compiler) stepWorkdir(container *yaml.Container) string {
+	return path.Join(c.base, c.path, container.Directory)
```

Comment:
Shouldn't it use filepath.Join?

Also currently it's not possible to provide absolute container directory

Reply:
But users will use them!

Reply:
and path trafesal is not an issue as you alwas could do it via commands anyway ...
so check if it is absolute the use this else use filepath.join i would say

Reply:
> An absolute path wasn't the idea behind it.

Yes I know this was just a suggestion that it would be nice to have otherwise it could be confusing for users